linter/package.json

```diff
@@ -7,7 +7,10 @@
   "private": false,
   "scripts": {
     "build-ts": "tsc",
-    "run-linter": "node dist/index.js"
+    "prerun-linter": "yarn run build-ts",
+    "run-linter": "node dist/index.js",
+    "pregen-schema": "yarn run build-ts",
+    "gen-schema": "node dist/bin/genSchema.js"
```

Comment:
Can you mention in the linter README about this command? otherwise lgtm

Reply:
Actually they are not real linter commands. They just allow us to run `yarn run-linter` without have to run `yarn built-ts` first. `npm` would run `pre<script>` before and `post<script>` after run `<script>` command (See the [docs](https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts)) and classic `yarn` inherits this benhaviour.
Alternative we can use [`ts-node`](https://www.npmjs.com/package/ts-node) to achieve the same goal.

Reply:
The linter README doesn't seem to be the right place to host `gen-schema` as it is meant to be run by the repository owner. Any ideas?

Reply:
I think it's a ok place to put linter related, Repo authors will mostly interact with the root dir anyway.

Reply:
OK. I have updated the doc.